common/flavorization/wc_councillors.txt

```diff
@@ -0,0 +1,101 @@
+@special_prio = 400
```

Comment:
Why you deleted stromic titles?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/135871384-692a176a-4602-456c-aace-69726fc3db6b.png)

</details>

Reply:
Remerod suggestion

Reply:
> Remerod suggestion

He wasn't against keeping Empire of Arathor.

Reply:
Yes, I was Suggesting to keep the "Empire" name but change the Emperor/Empress Title to High King/Queen

Reply:
So, just delete them. They are replaced by normal humans titles

Reply:
but won't be high Kingdom them? because i think Empire fits, just that the rulers should be High kings?

Reply:
The Empire of Arathor remains the Empire of Arathor.
This kind of title should be definite but not on this PR.